apps/server/src/modules/tool/service/tool-validation.service.ts

```diff
@@ -98,6 +103,17 @@ export class ToolValidationService {
 		});
 	}
 
+	private validateDefault(customParameter: CustomParameterDO[]): boolean {
```

Comment:
validateDefault sounds like an default Validation :D 
I think we should call it validateDefaultValueWithRegex or validateDefaultValue